src/js/shield_defs.js

```diff
@@ -3385,6 +3385,10 @@ export function loadShields(shieldImages) {
       bottom: 7,
     },
   };
+  shields["IN:NE"] = {
+    ...shields["IN:NH"],
+    numberingSystem: "roman",
```

Comment:
This touches on #465, but I think the NE shields are different than the other cases in that issue. It seems to be common to refer to the expressways by [European numerals](https://en.wikipedia.org/wiki/Expressways_of_India#National_Expressways) in writing, but shields or the gantry name signs mentioned in https://github.com/ZeLonewolf/openstreetmap-americana/issues/225#issuecomment-1375884357 mostly use Roman numerals. (I’ve come across photos of a few signs that use European numerals, but only as part of an [entrance portal](https://www.shutterstock.com/vi/image-photo/delhi-india-june-18-2022-welcome-2169899597).)